airflow/providers/salesforce/hooks/salesforce.py

```diff
@@ -74,6 +91,29 @@ def get_connection_form_widgets() -> Dict[str, Any]:
                 lazy_gettext("Security Token"), widget=BS3PasswordFieldWidget()
             ),
             "extra__salesforce__domain": StringField(lazy_gettext("Domain"), widget=BS3TextFieldWidget()),
+            "extra__salesforce__consumer_key": StringField(
+                lazy_gettext("Consumer Key"), widget=BS3TextFieldWidget()
+            ),
+            "extra__salesforce__private_key_file_path": PasswordField(
+                lazy_gettext("Private Key File Path"), widget=BS3PasswordFieldWidget()
+            ),
+            "extra__salesforce__private_key": PasswordField(
+                lazy_gettext("Private Key"), widget=BS3PasswordFieldWidget()
+            ),
+            "extra__salesforce__organization_id": StringField(
+                lazy_gettext("Organization ID"), widget=BS3TextFieldWidget()
+            ),
+            "extra__salesforce__instance": StringField(lazy_gettext("Instance"), widget=BS3TextFieldWidget()),
+            "extra__salesforce__instance_url": StringField(
+                lazy_gettext("Instance URL"), widget=BS3TextFieldWidget()
+            ),
+            "extra__salesforce__proxies": StringField(lazy_gettext("Proxies"), widget=BS3TextFieldWidget()),
+            "extra__salesforce__version": StringField(
+                lazy_gettext("Salesforce API Version"), widget=BS3TextFieldWidget()
```

Comment:
```suggestion
                lazy_gettext("API Version"), widget=BS3TextFieldWidget()
```
This is salesforce connection, just calling it api version is enough i believe